### PR TITLE
feat: use behavior ID only

### PR DESCRIPTION
**Pull request by LordMidas**
This is accomplished via a hook on root_state onInit function that populates an ID to Script file name map. This way we don't need the user to pass both the id and the script file name for the behavior.

### CODE REVIEW

**Review by TaroEld — APPROVED** [review]

File: scripts/config/msu/ai.nut [hunk 1]
```diff
@@ -1,4 +1,21 @@
...
+
+	function getBehaviorScriptFromID( _id )
+	{
+		if (this.m.BehaviorIDToScriptMap.len() == 0)
+		{
+			throw "This function cannot be used before the root_state.onInit function has run";
+		}
+		if (!(_id in this.m.BehaviorIDToScriptMap))
+		{
+			::logError("Invalid behavior id '" + _id + "' and/or the associated script file is not located in \'scripts/ai/tactical/behaviors\'");
```
Comment (Enduriel) [inline scripts/config/msu/ai.nut:12]: unescaped `'` around _id, but escaped `\'` around `scripts/ai/tactical/behaviors`. Not certain which one is the right one (pretty sure it's the first though), but we definitely shouldn't have 2 different ones.

**Review by Enduriel — APPROVED** [review]
Other than that small thing lgtm

File: scripts/config/msu/ai.nut (hunk 1 above)
Comment (LordMidas) [reply scripts/config/msu/ai.nut:12]: In my experience squirrel doesn't care but I still use the escape characters *shrug*. We can decide what convention we want to follow.

Comment (LordMidas) [reply scripts/config/msu/ai.nut]: We decided to go with escape characters as that is what the decompiler does for single quotes.

**Pull request merged**